docs/pages/versions/unversioned/sdk/bar-code-scanner.md

```diff
@@ -66,22 +65,30 @@ export default class BarcodeScannerExample extends React.Component {
       return <Text>No access to camera</Text>;
     }
     return (
-      <View style={{ flex: 1 }}>
+      <View style={{ flex: 1, flexDirection: 'column', justifyContent: 'flex-end'}}>
```

Comment:
`flexDirection` is default `column`, so that's a redundant style here I'm afraid?

Reply:
thanks for pointing that out!